src/com/animation.js

```diff
@@ -173,13 +176,26 @@
 			return animation;
 		};
 
+		/**
+		 * Get the renderer instence used for this specific animation
+		 * @returns renderer instence
+		 */
+		var getRenderer = function () {
+			return renderer;
+		};
+
 		/**
 		 * Set the ctn of the animation and call render if everything is ready
 		 * @param {jQuery Element} scope the container of the animation
+		 * @param {function} cb callback when the render is done
 		 */
-		var init = function (scope) {
+		var init = function (scope, opts, cb) {
```

Comment:
Please make the `cb` part of the opts object, so the signature will be `(scope, opts)`

I would name the property `onLoaded` or `onReady`

Reply:
@nitriques done.